tests/test_viz.py

```diff
@@ -303,6 +303,24 @@ def test_plot_heatmap_plots_all_nan_samples_with_nans(ocx, api_data, is_onecodex
     assert chart.encoding.x.sort[-1] == all_nan_sample.filename
 
 
+def test_plot_heatmap_with_haxis_two_cluster_groups(ocx, api_data):
+    import numpy as np
+
+    samples = ocx.Samples.where(project="4b53797444f846c4")
+    sample1 = ocx.Samples.get("cc18208d98ad48b3")
```

Comment:
Do we actually need all that new test data? We could probably set one of the existing sample's abundances in that project to `np.nan` to reproduce the bug

Reply:
@jairideout there are branches in the code that only execute if there are >3 samples so we had to add some